dashboard/src2/pages/LoginSignup.vue

```diff
@@ -218,6 +249,12 @@ export default {
 		}
 	},
 	methods: {
+		isOauthLogin() {
+			return this.oauthEmailDomain === null;
+		},
+		oauthProvider(emailDomain) {
+			return oauthProviders()[emailDomain].provider_name;
```

Comment:
Maybe a better way is to make a helper function in `logins.js` called `getOauthProvider(email)`